particula/util/vapor_flux.py

```diff
@@ -0,0 +1,52 @@
+""" calculate the vapor flux to the surface of particles
+"""
+
+import numpy as np
+from particula import u
+from particula.util.fuchs_sutugin import fsc as fsc_func
+from particula.util.input_handling import (in_concentration, in_handling,
+                                           in_scalar)
+from particula.util.molecular_enhancement import mol_enh
+from particula.util.particle_surface import area
+from particula.util.rms_speed import cbar
+
+
+def phi(  # pylint: disable=too-many-arguments
+    particle_area=None,
+    molecular_enhancement=None,
+    vapor_attachment=1,
```

Comment:
is vapor attachment, the same as the accommodation coefficent?

Reply:
In short, yes, but we should generally distinguish between different attachment coefficients. There are gonna be particle--particle attachment coefficients, and this is the vapor--particle coefficient. Once we start doing "chemistry" later on, these may need some more thought...